ipydatagrid/datagrid.py

```diff
@@ -378,18 +378,18 @@ def data(self):
     @staticmethod
     def generate_data_object(dataframe, guid_key="ipydguuid"):
         dataframe[guid_key] = pd.RangeIndex(0, dataframe.shape[0])
-        
-        # Renaming default index name from 'index' to 'id' on 
+
+        # Renaming default index name from 'index' to 'id' on
```

Comment:
Can we use something else, something that's not a reserved keywords in the language?

Reply:
Perhaps better to make it key? If it's internal detail we could do _key?

Reply:
so instead of having the index named `id`, have it named `key`?

Reply:
@gaborbernat would you be able to elaborate on why using 'id' as a string could be a problem?

Reply:
IMHO the key is just as informative, and if this is a pandas data frame, users could do a property lookup too, not just string lookup, not? Not a deal-breaker from my side, but I thought I'll flag key as a potential better name.